release/_publishing.sh

```diff
@@ -27,19 +27,18 @@ function add_fixed_issues_to_patcher_project_version {
 
 		IFS=',' fixed_issues="${fixed_issues_array[*]:start_index:fixed_issues_array_part_length}"
 
-		if (curl \
+		local update_fixed_issues_response=$(curl \
```

Comment:
@kiwm, can you please invoke the function to add a project version in Patcher to be the last one executed in the release_gold?